lightning/Cargo.toml

```diff
@@ -39,6 +39,7 @@ secp256k1 = { version = "0.20.2", default-features = false, features = ["alloc"]
 hashbrown = { version = "0.11", optional = true }
 hex = { version = "0.3", optional = true }
 regex = { version = "0.1.80", optional = true }
+backtrace = { version = "0.3", optional = true }
```

Comment:
Could this be a `dev-dependency`?

Reply:
Sadly, rustc says `dev-dependencies are not allowed to be optional`